import-scripts/study_directory_uses_updated_filenames.py

```diff
@@ -0,0 +1,538 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+
+def CANCER_STUDY_TAG():
```

Comment:
could we just define this as a string?

Reply:
actually similar comment for all the constants

Reply:
done